x-pack/plugin/esql/src/internalClusterTest/java/org/elasticsearch/xpack/esql/action/ForkIT.java

```diff
@@ -350,32 +350,6 @@ public void testScoringKeepAndSort() {
         }
     }
 
-    public void testRrf() {
```

Comment:
this test failed because we don't load the `EsqlPluginWithEnterpriseOrTrialLicense` test plugin.
I did not see a need to load this plugin for the FORK tests so I took this test out in its own `RrfIT` since it was testing RRF anyway.